### PR TITLE
Adding more flags

### PR DESCRIPTION
**Pull request by GabubuAvailable**
Adding more flags for the apps which are compatible JUST with one system.

Flags:
- wii_only
- vwii_only
- wii_mini_only

### CODE REVIEW

File: hbb/routes.py [hunk 1]
```diff
@@ -44,6 +44,15 @@ def apps_list():
...
                 long_description_prefix += "[CAUTION! Writes to NAND!] "
+            # and if "wii_only" flag is specified, prepend warning to long description
+            if "wii_only" in flags:
+                long_description_prefix += "[CAUTION! Compatible just with the Wii!] "
+            # and if "vwii_only" flag is specified, prepend warning to long description
+            if "vwii_only" in flags:
+                long_description_prefix += "[CAUTION! Compatible just with the vWii!] "
+            # and if "wii_mini_only" flag is specified, prepend warning to long description
+            if "wii_mini_only" in flags:
+                long_description_prefix += "[CAUTION! Compatible just with the Wii mini!] "
```
Comment (noahpistilli) [inline hbb/routes.py]: Should the strings have newlines?

Comment (GabubuAvailable) [reply hbb/routes.py]: Could you give me an example please?

**Comment by Artuto** [comment]
I am concerned by the length of this prefix, as app descriptions will be cut off by the HBB after a certain length. Maybe it should be shorter?

**Comment by GabubuAvailable** [comment]
> I am concerned by the length of this prefix, as app descriptions will be cut off by the HBB after a certain length. Maybe it should be shorter?

I've shorted the prefixes as much as I could. Could you re-check please?

File: hbb/routes.py [hunk 2]
```diff
@@ -44,6 +44,15 @@ def apps_list():
             long_description_prefix = ""
             if "writes_to_nand" in flags:
                 long_description_prefix += "[CAUTION! Writes to NAND!] "
+            # if "wii_only" flag is specified, prepend warning to long description
+            if "wii_only" in flags:
+                long_description_prefix += "[Wii only app!] "
```
Comment (Artuto) [inline hbb/routes.py]:
```suggestion
                long_description_prefix += "[Wii only] "
```

File: hbb/routes.py [hunk 3]
```diff
@@ -44,6 +44,15 @@ def apps_list():
...
+            # if "vwii_only" flag is specified, prepend warning to long description
+            if "vwii_only" in flags:
+                long_description_prefix += "[vWii only app!] "
```
Comment (Artuto) [inline hbb/routes.py]:
```suggestion
                long_description_prefix += "[vWii only] "
```

File: hbb/routes.py [hunk 4]
```diff
@@ -44,6 +44,15 @@ def apps_list():
...
+            # if "wii_mini_only" flag is specified, prepend warning to long description
+            if "wii_mini_only" in flags:
+                long_description_prefix += "[Wii mini only app!] "
```
Comment (Artuto) [inline hbb/routes.py]:
```suggestion
                long_description_prefix += "[Wii mini only] "
```

**Review by Artuto — CHANGES_REQUESTED** [review]

**Review by GabubuAvailable — COMMENTED** [review]
Sorry for this delay. I did commit your requests. Could you double-check it?

**Review by Artuto — APPROVED** [review]

**Pull request merged**